std/experimental/allocator/building_blocks/bitmapped_block.d

```diff
@@ -350,12 +350,12 @@ struct BitmappedBlock(size_t theBlockSize, uint theAlignment = platformAlignment
     `Ternary.no` otherwise. Never returns `Ternary.unkown`. (This
     method is somewhat tolerant in that accepts an interior slice.)
     */
-    Ternary owns(void[] b) const
+    pure nothrow @trusted @nogc
+    Ternary owns(const void[] b) const
     {
-        //if (!b.ptr) return Ternary.no;
-        assert(b.ptr !is null || b.length == 0, "Corrupt block.");
-        return Ternary(b.ptr >= _payload.ptr
-            && b.ptr + b.length <= _payload.ptr + _payload.length);
+        assert(b || b.length == 0, "Corrupt block.");
+        return Ternary(b && _payload && (&b[0] >= &_payload[0])
```

Comment:
If you made it trusted you can keep the previous implementation, which is marginally better on the eyes.